packages/langium/src/node/node-module-context.ts

```diff
@@ -0,0 +1,19 @@
+/******************************************************************************
+ * Copyright 2022 TypeFox GmbH
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License, which is available in the project root.
+ ******************************************************************************/
+
+import { DefaultSharedModuleContext } from '../default-module';
+import { createConnection, ProposedFeatures } from 'vscode-languageserver/node';
+import { NodeFileSystemProvider } from './node-file-system-provider';
+
+export function createNodeModuleContext(context: DefaultSharedModuleContext = {}): DefaultSharedModuleContext {
```

Comment:
This method is specifically for a language server context. Its name does not reflect this.